src/flaskr/app/modules/api/session/admin/delete.py

```diff
@@ -0,0 +1,41 @@
+from flask import jsonify, Flask
+from ....resources.reset_token import RevokedTokenModel
+from flask_restplus.namespace import RequestParser
+from flask_jwt_extended import ( jwt_required, get_raw_jwt )
+
+#################
+# Parser        #
+#################
+Parser = RequestParser()
+Parser.add_argument('Authorization',
+                    help='Token of the admin.',
+                    required=True,
+                    location='headers')
+
+#################
+# Method        #
+#################
+
+
+@jwt_required
+def Delete(args):
+    '''
+    jti = unique identifier of the token

```

Comment:
Here should be a description of the method, something along the lines "Log off an admin"